spring-cloud-modules/spring-cloud-open-telemetry/spring-cloud-open-telemetry1/src/test/java/com/baeldung/opentelemetry/SpringContextTest.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.opentelemetry;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = ProductApplication.class)
+public class SpringContextTest {
+    
+    @Test
+    public void whenSpringContextIsBootstrapped_thenNoExceptions() {
```

Comment:
JUnit 5 test functions and classes don't use the `public` modifier - we can remove this and leave them at default visibility.

Reply:
removed public